---
 bindgen/src/builder.rs | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/bindgen/src/builder.rs b/bindgen/src/builder.rs
index bc18406..db710b3 100644
--- a/bindgen/src/builder.rs
+++ b/bindgen/src/builder.rs
@@ -44,7 +44,7 @@ impl Builder {

     pub fn bindgen_builder(&self) -> bindgen::Builder {
         // Begin building the bindgen params.
-        let mut builder = bindgen::Builder::default();
+        let mut builder = bindgen::Builder::default().detect_include_paths(false);

         let mut clang_args = vec!["-x", "objective-c", "-fblocks", "-fmodules"];
         let target_arg;
--
2.43.2

